pom.xml

```diff
@@ -0,0 +1,101 @@
+
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+        <maven-surefire-plugin.suiteXmlFile>
+            ${project.build.testOutputDirectory}/testng/runAllTests.xml
+        </maven-surefire-plugin.suiteXmlFile>
+
+        <testng.version>7.4.0</testng.version>
+        <assertj-core.version>3.4.1</assertj-core.version>
+        <maven-surefire-plugin.version>3.0.0-M5</maven-surefire-plugin.version>
+        <maven-compiler-plugin.version>3.8.0</maven-compiler-plugin.version>
+    </properties>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>${maven-surefire-plugin.version}</version>
+<!--                <configuration>-->
+<!--&lt;!&ndash;                    <suiteXmlFiles>&ndash;&gt;-->
+<!--&lt;!&ndash;                        <suiteXmlFile>${maven-surefire-plugin.suiteXmlFile}</suiteXmlFile>&ndash;&gt;-->
+<!--&lt;!&ndash;                    </suiteXmlFiles>&ndash;&gt;-->
+<!--                </configuration>-->
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>${maven-compiler-plugin.version}</version>
+                <configuration>
+                    <source>1.8</source>
+                    <target>1.8</target>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+
+    <profiles>
+        <profile>
+            <id>runAddAndSubtractTests</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <properties>
+                <maven-surefire-plugin.suiteXmlFile>
+                    ${project.build.testOutputDirectory}/testng/runAddAndSubtractTests.xml
+                </maven-surefire-plugin.suiteXmlFile>
+            </properties>
+        </profile>
+        <profile>
+            <id>runAllTests</id>
+            <properties>
+                <maven-surefire-plugin.suiteXmlFile>
+                    ${project.build.testOutputDirectory}/testng/runAllTests.xml
+                </maven-surefire-plugin.suiteXmlFile>
+            </properties>
+        </profile>
+        <profile>
+            <id>runMultiplyAndDivideTests</id>
+            <properties>
+                <maven-surefire-plugin.suiteXmlFile>
+                    ${project.build.testOutputDirectory}/testng/runMultiplyAndDivideTests.xml
+                </maven-surefire-plugin.suiteXmlFile>
+            </properties>
+        </profile>
+    </profiles>
+
+    <dependencies>
+        <dependency>
+            <groupId>calculator</groupId>
+            <artifactId>calculator</artifactId>
+            <scope>system</scope>
+            <version>1.0</version>
+            <systemPath>${project.basedir}/lib/calculator-1.0.jar</systemPath>
```

Comment:
Looks like you forgot to add `lib` directory to repository

Reply:
I got it on the local machine but it turned green after a I had made a commit so I do not know why it did not go to the repository. Do you have any ideas why?

Reply:
done